src/main/groovy/com/morpheusdata/hyperv/HyperVApiService.groovy

```diff
@@ -615,7 +616,7 @@ class HyperVApiService {
             def out = wrapExecuteCommand(command, opts)
             log.debug("out: ${out.data}")
             if (out.success) {
-                hasMore = out.data != ''
+                hasMore = out.data != null
```

Comment:
this should be '' or null

Reply:
review comment changes added as suggested